.git-commit-template

```diff
@@ -2,8 +2,52 @@ COMPONENT: Subject
 
 Explanation
 
-Resolves:
-https://github.com/SSSD/sssd/issues/XXXX
+Resolves: https://github.com/SSSD/sssd/issues/XXXX
+
+# If a release note is required, choose one of the tags and place it here.
```

Comment:
"one and **only** one" tag, right?

Reply:
Or will automation pickup all tags found?

Reply:
It will pick up all tags found as there may be cases where one commit requires release notes in multiple categories. E.g. new feature that also adds new package and new options in one commit - you need to describe the feature, packaging changes and options.

(Of course this particular example should be split into three commits)

Reply:
I added a note about it.

Reply:
Then I would replace "one" with "appropriate" or "corresponding"